docs/integrations/development.md

```diff
@@ -1526,3 +1526,63 @@ immediately reflect that in the graph.
 We hope to provide a good interface via this SDK for providing an interface for
 handling events via this SDK and providing commands/utilities for testing events
 to get an understanding of how they may affect the JupiterOne graph.
+
+## j1 CLI
```

Comment:
Might be better to start on line 1490 with the other j1 cli info. Up to you.

Reply:
To be honest, I went back and forth with putting it higher up starting at line 1490.  My main reason for leaving it at the bottom is it felt like it was interrupting the `j1-integration` documentation too much when placed up there.

I'll leave it where it is for now, but we can move it if we start to see its location confusing people.